docs/uniques.md

```diff
@@ -23,17 +23,17 @@ Applicable to: Global, FollowerBelief, Improvement
 #### [stats] [cityFilter]
 Example: "[+1 Gold, +2 Production] [in all cities]"
 
-Applicable to: Global
+Applicable to: Global, FollowerBelief
```

Comment:
The changes to uniques.md are completely additional and are not really a part of the nuke revamp right? I've got no clue how the uniques work as I've never worked with them.

Reply:
Jup, this is an automatically generated documentation file. It should have been updated as part of #5882, but I forgot to include the file then, so it's included here instead